pkgs/top-level/python-packages.nix

```diff
@@ -7851,7 +7851,9 @@ in {
 
   translationstring = callPackage ../development/python-modules/translationstring { };
 
-  transmissionrpc = callPackage ../development/python-modules/transmissionrpc { };
+  transmission-rpc = callPackage ../development/python-modules/transmission-rpc { };
+
+  transmissionrpc = self.transmission-rpc; # alias for compatibility 2020-02-07
```

Comment:
```suggestion
  transmission-rpc = callPackage ../development/python-modules/transmission-rpc { };
  transmissionrpc = self.transmission-rpc; # alias for compatibility 2020-02-07
```

Reply:
This is not convention to remove the newline in between, see e.g. dnspython, etc. in the same file.